src/index.js

```diff
@@ -125,7 +126,9 @@ class HMIApp extends React.Component {
     }
 
     changeResetPeriod(event) {
-        store.dispatch(resetTimeout(event.target.value));   
+        if (event.target.value > OUT_OF_BOUND_RESET_PERIOD) {
```

Comment:
@LiliiaShlikhtLuxoft @YuriiYefimenko Seems check is incorrect? Should be `event.target.value > OUT_OF_BOUND_RESET_PERIOD`. In other case input value should be replaced with default. And check `OUT_OF_BOUND_RESET_PERIOD` should be erased from other place

Reply:
@ValeriiMalkov fixed